modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/genitals.dm

```diff
@@ -12,19 +12,37 @@
 	if(!badonkers)
 		return TRUE
 	switch(badonkers.visibility_preference)
-		if(GENITAL_ALWAYS_SHOW)
+		if(GENITAL_ALWAYS_SHOW) //Never hidden
 			return FALSE
-		if(GENITAL_HIDDEN_BY_CLOTHES)
+		if(GENITAL_HIDDEN_BY_CLOTHES) //Hidden if the relevant body parts are covered by clothes or underwear
+			//Do they have a Uniform or Suit that covers them?
 			if((H.w_uniform && H.w_uniform.body_parts_covered & genital_location) || (H.wear_suit && H.wear_suit.body_parts_covered & genital_location))
```

Comment:
Since you're already there, can you also get rid of the single-letter variables (or double letters, for `HD`)?